conf/test_full.config

```diff
@@ -15,9 +15,17 @@ params {
     config_profile_description = 'Full test dataset to check pipeline function'
 
     // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
+    input           = 'https://raw.githubusercontent.com/nf-core/test-datasets/isoseq/samplesheet_full.csv'
+    primers         = 'https://raw.githubusercontent.com/nf-core/test-datasets/isoseq/testdata/primers_complete.fasta'
+    fasta           = 'http://ftp.ensembl.org/pub/release-106/fasta/sus_scrofa/dna/Sus_scrofa.Sscrofa11.1.dna.toplevel.fa.gz'
+    gtf             = 'http://ftp.ensembl.org/pub/release-106/gtf/sus_scrofa/Sus_scrofa.Sscrofa11.1.106.gtf.gz'
+    rq              = 0.9
+    chunk           = 100
+    five_prime      = 100
+    splice_junction = 10
+    three_prime     = 100
+    capped          = true
+    aligner         = 'ultra'
 
     // Genome references
     genome = 'R64-1-1'
```

Comment:
The advantage of using igenomes for the AWS tests as well is that the `igenomes` bucket will be mounted on the compute instance and doesn't need to be downloaded again at runtime

Reply:
I just check the igenome database on our cluster . 
The Pig genome is absent, it could be the cause?

Reply:
Where this is set up is in the `main.nf` file. There you have this line:
https://github.com/nf-core/isoseq/blob/963e0000599cdce0d48e6530c34da5dd421498c8/main.nf#L20

You can also add the `gtf` reading below this as follows:
```
params.gtf = WorkflowMain.getGenomeAttribute(params, 'gtf')
```

But for the `fasta` file this should already be setup.

Reply:
> I just check the igenome database on our cluster . The Pig genome is absent, it could be the cause?

Yes, that is likely the cause. You could try to sync the igenome database on your cluster, or provide `--igenome_base "s3://ngi-igenomes/igenomes"` which would pull again the references directly from igenomes. Does this work for you?

Reply:
I made some tests, even with `--igenome_base "s3://ngi-igenomes/igenomes"`, this error is still here. 
So I took a look to how the parameters are set. 
When I print `val` from `getGenomeAttribute` in `lib/WorkflowMain.groovy`, the data path is resolved correctly: 
`VAL => s3://ngi-igenomes/igenomes/Sus_scrofa/Ensembl/Sscrofa10.2/Sequence/WholeGenomeFasta/genome.fa`

When I print the result of  `getGenomeAttribute` from `main.nf`, I get `null`. 

It have no sense to me 😵

Reply:
Removing default values from nextflow.config solved the problem. 
The AWS test finished without errors.